src/thumbnailFactory.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var ImageResource = require('./ImageResource');
+
+var _getThumbService = function(image, width) {
```

Comment:
I guess my comment about where this should go in #102 is moot, but the naming comment still holds.